### ISSUE TITLE
npm start failed with Error: resource temporarily unavailable

### ISSUE THREAD

**Issue by dfang** [ipfs-shipyard/ipfs-desktop#724]
Please describe what you were doing when this error happened.

**Specifications**

- **OS**: darwin
- **IPFS Desktop Version**: 3.0.10
- **Electron Version**: 3.0.10
- **Chrome Version**: 66.0.3359.181

**Error**

```
Error: Command failed: /Users/mj/GitHub/ipfs-desktop/node_modules/go-ipfs-dep/go-ipfs/ipfs daemon --migrate=true --routing=dhtclient
Error: resource temporarily unavailable

Initializing daemon...
go-ipfs version: 0.4.18-
Repo version: 7
System version: amd64/darwin
Golang version: go1.11.1
Received interrupt signal, shutting down...
(Hit ctrl-c again to force-shutdown the daemon.)

    at makeError (/Users/mj/GitHub/ipfs-desktop/node_modules/execa/index.js:174:9)
    at Promise.all.then.arr (/Users/mj/GitHub/ipfs-desktop/node_modules/execa/index.js:278:16)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by lidel** [comment]
hey @dfang I think the error is caused by a stale lockflile.

Automatic cleanup is being prepared in https://github.com/ipfs-shipyard/ipfs-desktop/pull/722 but until that lands, try:
- running `ipfs repo fsck` 
- or manually removing `api` and `repo.lock` from your `$IPFS_PATH`

.. and starting ipfs-desktop again.

Let me know if this helped.

**Comment by lidel** [comment]
@dfang this should be fixed in [v0.6.0](https://github.com/ipfs-shipyard/ipfs-desktop/releases/tag/v0.6.0)
